### PR TITLE
Feature/pluralization

### PR DESCRIPTION
**Pull request by macbookandrew**
This fixes the case where there is only one row/result for English.

### CODE REVIEW

File: resources/views/bootstrap-4/includes/bulk-select-row.blade.php [hunk 1]
```diff
@@ -23,7 +23,7 @@ class="btn btn-primary btn-sm"
                     <span>
                         @lang('You are currently selecting all')
                         <strong>{{ number_format($rows->total()) }}</strong>
-                        @lang('rows').
```
Comment (rappasoft) [inline resources/views/bootstrap-4/includes/bulk-select-row.blade.php:26]: Wouldn't this be 'rows' regardless?

File: resources/views/tailwind/includes/pagination.blade.php [hunk 2]
```diff
@@ -1,12 +1,20 @@
 @if ($showPagination)
     <div class="p-6 md:p-0">
         @if ($paginationEnabled && $rows->lastPage() > 1)
-            {{ $rows->links() }}
```
Comment (rappasoft) [inline resources/views/tailwind/includes/pagination.blade.php:4]: Why are we replacing this?

File: resources/views/bootstrap-4/includes/bulk-select-row.blade.php (hunk 1 above)
Comment (rappasoft) [reply resources/views/bootstrap-4/includes/bulk-select-row.blade.php:26]: I also feel like [these](https://github.com/rappasoft/laravel-livewire-tables/pull/415/files#diff-f7f9c78938d95da7404fbfe23bc215ae70b57ce1b0eb3bcc3a481e7f3716879dR9) if anything should have been replaced, the rest of the changes don't seem too worth it to me.

**Comment by rappasoft** [comment]
Needs work, i'll handle.

File: resources/views/bootstrap-4/includes/bulk-select-row.blade.php (hunk 1 above)
Comment (andrewminion-luminfire) [reply resources/views/bootstrap-4/includes/bulk-select-row.blade.php:26]: Oh, I guess this is only used if there’s more than one page.

File: resources/views/tailwind/includes/pagination.blade.php (hunk 2 above)
Comment (andrewminion-luminfire) [reply resources/views/tailwind/includes/pagination.blade.php:4]: The tailwind paginator lacks pluralization support.

**Review by andrewminion-luminfire — COMMENTED** [review]
Ok; thanks!

**Comment by macbookandrew** [comment]
Sorry…commented with my work account instead of personal account.

**Pull request closed without merging**